### ISSUE TITLE
Can't install the module

### ISSUE THREAD

**Issue by keivan85** [cadencelabs/urldedup#1]
I tried to install the module via composer(I created a composer file for the module), but when I try to run bin/magento setup:di:compile and error is shown.

It show MiBPHP Fatal error:  Class 'Cadence\UrlDedup\Console\Command\AbstractCatalogUrlRewrites' not found in /var/www/MY-SITE/vendor/cadence/urldedup/Console/Command/CategoryUrlRewrites.php on line 19
is it something that can be fixed?

**Comment by cadencelabs-master** [comment]
We don't have a composer package for the module, follow the installation instructions here: https://www.cadence-labs.com/2017/11/magento-2-fix-duplicate-url-key-for-categories-and-products/

**Comment by keivan85** [comment]
I tried your installation on first place but it didn't work for me. I have all my modules and theme installed with composer. I tried to run your stated command
mkdir -p app/code/Cadence/UrlDedup && git clone https://github.com/cadencelabs/urldedup app/code/Cadence/UrlDedup
The repo got copied to my project. Then I ran upgrade and di:compile command & flush:cache, but your module didn't show up in module list.
